Packs/Intezer/Integrations/IntezerV2/IntezerV2.py

```diff
@@ -188,7 +191,8 @@ def get_latest_result_command(intezer_api: IntezerApi, args: Dict[str, str]) ->
     if not latest_analysis:
         return _get_missing_file_result(file_hash)
 
-    return enrich_dbot_and_display_file_analysis_results(latest_analysis.result())
+    file_metadata = latest_analysis.get_root_analysis().metadata
```

Comment:
Shouldn't the metadata() function be called?
```suggestion
    file_metadata = latest_analysis.get_root_analysis().metadata()
```

Reply:
metadata is a python property

Reply:
Yes it is clear
But I think metadata is `None` until the `metadata()` function is called,
I'm wrong?

Reply:
Python behind the scene is calling the function, properties in Python are "sugar syntax"

Reply:
You're right, I missed it